bundler/spec/dependabot/bundler/update_checker/version_resolver_spec.rb

```diff
@@ -455,7 +432,7 @@
 
           it "ignores the minimum ruby version in the gemspec" do
             expect(resolver.latest_resolvable_version_details[:version]).
-              to eq(Gem::Version.new("7.2.0"))
+              to eq(Gem::Version.new("7.4.0"))
```

Comment:
I think this version number change was a naieve 'fix' for us by-passing stubs and hitting the internet for real.

We should probably walk this back and move this test into the native spec.

Reply:
👍  thoughts on keeping it here for now and adding a TODO to move to native helper spec? Could do another pass at specs when we add bundler 2.